Checkout/Source/Tokenisation/CheckoutAPIService.swift

```diff
@@ -170,3 +171,44 @@ final public class CheckoutAPIService: CheckoutAPIProtocol {
     }
   }
 }
+
+// MARK: Security Code Request
+
+extension CheckoutAPIService {
+  public func createSecurityCodeToken(securityCode: String, completion: @escaping (Result<SecurityCodeResponse, TokenisationError.SecurityCodeError>) -> Void) {
```

Comment:
For public functions, Just wondering do you need to add any comments for documentation ?

Reply:
This has been discussed offline and we wanted to follow the path of the `createToken` function in the same file since we don't encourage the usage of this function in headless mode but still have to define it as `public` since we need to expose it to `Frames`. (This function lives under `Checkout` which is another module) @deepesh-vasthimal-cko

Reply:
resolving this as I believe the offline discussion decided to keep it this way. feel free to re-open if it doesn't feel right